src/SerilogWeb.Classic/Classic/ApplicationLifecycleModule.cs

```diff
@@ -159,7 +159,9 @@ public void Init(HttpApplication application)
                         return;
 
                     var error = application.Server.GetLastError();
-                    var level = error != null ? LogEventLevel.Error : _requestLoggingLevel;
+                    var level = error != null || 
+                        (application.Response.StatusCode >= 400 && application.Response.StatusCode <= 599) 
```

Comment:
I think currently we consider 400-499 to be non-exceptional as they're client rather than server errors. With the exception logger in place, will `error` be non-null here? If so perhaps keeping the original logic is preferable?